package.json

```diff
@@ -56,7 +61,7 @@
     "@types/sinon": "^7.0.9",
     "@types/storybook__addon-actions": "^3.4.2",
     "@types/storybook__react": "^4.0.1",
-    "@types/styled-components": "^4.1.12",
+    "@types/styled-components": "4.1.13",
```

Comment:
Not exactly, I installed strict version 4.1.13, because I had problems with 4.1.12 in my IDE.
Will "^4.1.13" work or it's better to leave old version "^4.1.12".

Reply:
Ok